syntax = "proto3";

package com.example;

message StopStopwatchParams {
}

message StopStopwatchResult {
  double time_elapsed = 1;
}
